.github/ISSUE_TEMPLATE/support.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Support
-about: If you have questions about the Prometheus Operator
+about: If you have questions about the Prometheus Operator. If you have questions about helm installation, go to https://github.com/prometheus-community/helm-charts repository. If you have questions about kube-prometheus setup, go to https://github.com/prometheus-operator/kube-prometheus
```

Comment:
funnily enough many of the issues we used to see in kube-prometheus should also directed towards the helm-charts repo. This is also the case typically in the Slack channels.

Given that, I am not sure this will help greatly overall but have no objections to it either.